app/services/forest_liana/schema_utils.rb

```diff
@@ -2,9 +2,14 @@ module ForestLiana
   class SchemaUtils
 
     def self.associations(active_record_class)
-      active_record_class
-        .reflect_on_all_associations
-        .select { |association| !polymorphic?(association) && !is_active_type?(association.klass) }
+      associations = active_record_class.reflect_on_all_associations.select do |association|
+        next if association.options[:polymorphic] && !is_active_type?(association.klass)
+        begin
+          association.klass
```

Comment:
it probably works but a boolean is expected here too (implicit return of the select).